examples/graphics/material-physical-cubemap-rotation.html

```diff
@@ -0,0 +1,136 @@
+        app.root.addChild(camera);
+
+        // Load a cubemap asset. This DDS file was 'prefiltered' in the
+        // PlayCanvas Editor and then downloaded.
+        var cubemapAsset = new pc.Asset('helipad.dds', 'cubemap', {
+            url: "../assets/cubemaps/helipad.dds"
+        }, {
+            type: pc.TEXTURETYPE_RGBM
+        });
+        app.assets.add(cubemapAsset);
+        app.assets.load(cubemapAsset);
+        cubemapAsset.ready(function () {
+            app.scene.setSkybox(cubemapAsset.resources);
+        });
+
+        var NUM_SPHERES = 5;
+
+        var entity;
+
+        var createSphere = function (x, y, z) {
+            var material = new pc.StandardMaterial();
+            material.metalness = (x===2 && y===2) ? 1.0 : y / (NUM_SPHERES - 1);
+            material.shininess = (x===2 && y===2) ? 100 : x / (NUM_SPHERES - 1) * 100;
+            material.useMetalness = true;
+            material.update();
+
+            var sphere = new pc.Entity();
+            sphere.addComponent("model", {
+                material: material,
+                type: (x===2 && y===2) ? "box" : "sphere"
+            });
+            sphere.setLocalPosition(x - (NUM_SPHERES - 1) * 0.5, y - (NUM_SPHERES - 1) * 0.5, z);
+            sphere.setLocalScale(0.9, 0.9, 0.9);
+            app.root.addChild(sphere);
+            entity = (x===2 && y===2) ? sphere : entity;
+        };
+
+        var createText = function (fontAsset, message, x, y, z, rot) {
+            // Create a text element-based entity
+            var text = new pc.Entity();
+            text.addComponent("element", {
+                anchor: [0.5, 0.5, 0.5, 0.5],
+                fontAsset: fontAsset,
+                fontSize: 0.5,
+                pivot: [0.5, 0.5],
+                text: message,
+                type: pc.ELEMENTTYPE_TEXT
+            });
+            text.setLocalPosition(x, y, z);
+            text.setLocalEulerAngles(0, 0, rot);
+            app.root.addChild(text);
+        };
+
+        for (var i = 0; i < NUM_SPHERES; i++) {
+            for (var j = 0; j < NUM_SPHERES; j++) {
+                createSphere(j, i, 0);
+            }
+        }
+
+        // Load a font
+        var fontAsset = new pc.Asset('arial.json', "font", { url: "../assets/fonts/arial.json" });
+        fontAsset.on('load', function () {
+            createText(fontAsset, 'Glossiness', 0, -(NUM_SPHERES + 1) * 0.5, 0, 0);
+            createText(fontAsset, 'Metalness', -(NUM_SPHERES + 1) * 0.5, 0, 0, 90);
+        });
+        app.assets.add(fontAsset);
+        app.assets.load(fontAsset);
+
+        var mouse = new pc.Mouse(document.body);
+
+        var x = 0;
+        var y = 0;
+        var rot = new pc.Quat();
+
+        mouse.on('mousemove', function (event) {
+            if (event.buttons[pc.MOUSEBUTTON_LEFT]) {
+                x += event.dx;
+                y += event.dy;
+
+                // entity.setLocalEulerAngles(0.2 * y, 0.2 * x, 0);
```

Comment:
remove the comment?

Reply:
I shall remove this - and some other changes

Reply:
done - thanks